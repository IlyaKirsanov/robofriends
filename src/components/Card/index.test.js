import React from 'react'
import Card from './index'
import {shallow} from 'enzyme'


it('expect to render Card component', () =>{
	const card = shallow(<Card />)
	expect(card).toMatchSnapshot();
})