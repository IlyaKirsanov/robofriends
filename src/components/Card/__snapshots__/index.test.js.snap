// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="bg-light-green tc dib br3 pa3 ma3 grow b2 shadow-5"
>
  <img
    alt=""
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
